includes/class-sensei-course-structure.php

```diff
@@ -230,7 +230,8 @@ private function get_modules(): array {
 	 * @return bool|WP_Error
 	 */
 	public function save( array $raw_structure ) {
-		$structure = $this->sanitize_structure( $raw_structure );
+		$raw_structure = $this->ensure_modules_have_names( $raw_structure );
```

Comment:
Has this check become redundant now? https://github.com/Automattic/sensei/blob/53dbffabe6b43f4aa2724bfdfcf439ad6b7d50ff/includes/class-sensei-course-structure.php#L897-L902